sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientTestsLive.cs

```diff
@@ -0,0 +1,275 @@
+            TextAnalyticsClient client = GetClient();
+            string input = "Microsoft fue fundado por Bill Gates y Paul Allen.";
+
+            RecognizeEntitiesResult result = await client.RecognizeEntitiesAsync(input, "es");
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(3, entities.Count);
+        }
+
+        [Test]
+        public async Task RecognizeEntitiesWithSubtypeTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "I had a wonderful trip to Seattle last week.";
+
+            RecognizeEntitiesResult result = await client.RecognizeEntitiesAsync(input);
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(2, entities.Count);
+
+            foreach (NamedEntity entity in entities)
+            {
+                if (entity.Text == "last week")
+                    Assert.IsNotNull(entity.SubType);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "A developer with SSN 555-55-5555 whose phone number is 222-222-2222 is building tools with our APIs.";
+
+            RecognizePiiEntitiesResult result = await client.RecognizePiiEntitiesAsync(input);
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(2, entities.Count);
+
+            var entitiesList = new List<string> { "555-55-5555", " 222-222-2222 " };
+            foreach (NamedEntity entity in entities)
+            {
+                Assert.IsTrue(entitiesList.Contains(entity.Text));
+                Assert.IsNotNull(entity.Type);
+                Assert.IsNotNull(entity.Score);
+                Assert.IsNotNull(entity.Offset);
+                Assert.IsNotNull(entity.Length);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesWithLanguageTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "A developer with SSN 555-55-5555 whose phone number is 222-222-2222 is building tools with our APIs.";
+
+            RecognizePiiEntitiesResult result = await client.RecognizePiiEntitiesAsync(input, "en");
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(2, entities.Count);
+        }
+
+        [Test]
+        public async Task RecognizeLinkedEntitiesTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "Microsoft was founded by Bill Gates and Paul Allen.";
+
+            RecognizeLinkedEntitiesResult result = await client.RecognizeLinkedEntitiesAsync(input);
+
+            Assert.IsNotNull(result.Id);
+            Assert.AreEqual(3, result.LinkedEntities.Count);
+            Assert.IsNotNull(result.Statistics.CharacterCount);
```

Comment:
@annelo-msft when exploring the `RecognizeLinkedEntitiesResult ` object, it contains `Statistics` property. 
I couldn't find a way to set `IncludeStatistics` to true, which according to [documentation](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextDocumentStatistics.cs#L10), is the way to activate getting the statistics from the service.
Could you show me how to do it?

Reply:
This isn't available on the single-input method.  It can be set on the version of the method that takes multiple TextDocumentInput.

Reply:
I see. As a user, it is confusing than when exploring the `RecognizeLinkedEntitiesResult` object, you see the `Statistics` property.
Was this intended?

Reply:
It was, but I see your point.  Can you file an issue to track this?  We'll either need to update the docs or modify the API.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/9378